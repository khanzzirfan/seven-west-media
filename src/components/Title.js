import React from 'react';
import styled from 'styled-components';

const marginLeft = 10;
const fontSize = 20;

const Title = styled.h2`
    color: white;
    margin: 0 0 0 ${marginLeft}px;
    font-size: ${fontSize}px;
    font-family: Roboto, sans-serif;
    display: flex;
    justify-content: center;
    flex-direction: column;
    text-align: center;
`;

export default Title;