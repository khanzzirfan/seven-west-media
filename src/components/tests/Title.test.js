import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';

import Title from '../Title';

describe('components -> Title component', () => {
    it('should renders correctly', () => {
        const wrapper = TestRenderer.create(<Title>Media title</Title>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});