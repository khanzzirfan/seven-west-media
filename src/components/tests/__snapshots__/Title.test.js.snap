// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components -> Title component should renders correctly 1`] = `
.c0 {
  color: white;
  margin: 0 0 0 10px;
  font-size: 20px;
  font-family: Roboto,sans-serif;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  text-align: center;
}

<h2
  className="c0"
>
  Media title
</h2>
`;
